src/components/pacbio/PacbioRunPlate.vue

```diff
@@ -82,9 +92,26 @@ export default {
     onWellClick(position, plateNumber) {
       this.$refs.modal.showModalForPositionAndPlate(position, plateNumber)
     },
+    sequencingKitBoxBarcode(plateNumber) {
+      return this.runItem.plates[plateNumber].sequencing_kit_box_barcode
+    },
     // Only used for Revio runs
     serialNumber(plateNumber) {
-      return this.runItem.plates[plateNumber].sequencing_kit_box_barcode.substring(15, 20)
+      return this.sequencingKitBoxBarcode(plateNumber).substring(15, 20)
```

Comment:
Advisory: I am wondering if it would be worth moving the validation to the labware object itself? Happy to think about that as part of my refactor so as not to delay.

Reply:
Sounds sensible, but ye maybe worth doing once we have refactored to have the PlateList and Plate components, as sequenceingKitBoxBarcode may then be a data attribute and serialNumber a computer etc.

Reply:
Indeed. I will add it to the list.